content/akeneo-event-platform/platform-usage-requirements.md

```diff
@@ -0,0 +1,26 @@
+# Platform usage requirements
+
+
+<i class="fa fa-check-square"></i> If you want to use a **Pub/Sub destination**, you need to grant the Akeneo Event Platform publishing access. Follow these steps:
+
+1. In the Google Cloud Console, go to **Pub/Sub > Topics**.
+2. Next to the topic you want to use, click **...** and then **View permissions**.
+3. Click **ADD PRINCIPAL**.
+4. Paste the following service account address into the **New principals** text box: `delivery-sa@akecld-prd-sdk-aep-prd.iam.gserviceaccount.com`.
+5. In the Role drop-down list, select **Pub/Sub** and then **Pub/Sub Publisher**.
+6. Click **Save**.
```

Comment:
This is the third time we have this section (First time in the homepage, second one in the "Getting started").
I feel it's a bit too much and harder to maintain.